releng/com.zeligsoft.papyrus-cx.axcioma.rcp.product/rpm_build/papyrus-cx-axcioma.spec

```diff
@@ -0,0 +1,90 @@
+%define ifdef()       		%if %{expand:%%{?%{1}:1}%%{!?%{1}:0}}
+%define _default_version	2.2.0
+%define _default_release	0.%(date "+%Y%m%d%H%M")
+%define _rpmdir				%{_projectdir}/rpm_build/
+%define _targetdir			%{_projectdir}/target/
+%define _productdir			%{_targetdir}/products/
+%define __jar_repack		%{nil}
+
+# Set the release of the RPM according to the value of the 'parameterized' input
+# variables, or use a string constant
+%if "${_unqualified_version}" != "unknown"
+	%define version %{_unqualified_version}
+%else
+	%define version %{_default_version}
+%endif
+
+%if "%{_build_qualifier}" != "unknown"
+	%define release %{_build_qualifier}
+%else
+	%if 0%{?_git_tag:1}
+		%define release %{_default_release}_%{_git_tag}
+	%else
+		%if 0%{?_git_commit_id:1}
+			%define release %{_default_release}_%{_git_commit_id}
+		%else
+			%define release %{_default_release}
+		%endif
+	%endif
+%endif
+
+%global _productzip	papyrus-cx-axcioma-%{_full_version}-linux.gtk.x86_64.zip
+
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# Define packages
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+Name:          papyrus-cx-axcioma
+Version:       %{version}
+Release:       %{release}
```

Comment:
@eposse Now that we're building these RPMs for EL7 and EL8, could we please have `%{dist}` added to the release field? So something like...
`Release:       %{release}%{dist}`

`%{dist}` is a built-in RPM macro that evaluates to ".el7" or ".el8" on EL systems. That will help us keep those RPMs straight visually.

Reply:
@J17359 Sure. Since I already merged this, I'll open a new PR.

Reply:
@J17359 I just pushed the change directly to the papyrus branch, as the change was so minor.